### PR TITLE
Adding libm to src/meson.build

### PR DESCRIPTION
**Pull request by SolarAquarion**

### CODE REVIEW

File: src/meson.build [hunk 1]
```diff
@@ -43,6 +43,8 @@ finalterm_deps = [
   libutil,
 ]
 
+dep_m = cc.find_library('m')
```
Comment (jtojnar) [inline src/meson.build]: Could you please declare the dependency in top-level `meson.build` like `libutil` is, that seems to be the convention with Meson. Also please use `libm` for the variable name.

File: src/meson.build [hunk 2]
```diff
@@ -57,6 +59,6 @@ executable('finalterm', finalterm_sources + resources,
   include_directories: [ config_h_dir ],
   c_args: finalterm_c_args,
   vala_args: finalterm_vala_args,
-  dependencies: finalterm_deps,
+  dependencies: [finalterm_deps, dep_m],
```
Comment (jtojnar) [inline src/meson.build]: Could you add this to the `finalterm_deps` variable like other dependenies instead.

**Review by jtojnar — CHANGES_REQUESTED** [review]
Thanks for the PR, I will test on Arch Later.

**Comment by SolarAquarion** [comment]
I think i changed it as appropriate

**Comment by jtojnar** [comment]
Thank you.

**Pull request merged**